### ISSUE TITLE
Import rules: sub-entities are not settable

### ISSUE THREAD

**Issue by FRANCOIS50** [fusioninventory/fusioninventory-for-glpi#2507]
It not possible to set sub-entities in import rules If you want to set a criteria with an entity it is not possible to set a sub-entity.

**Comment by FRANCOIS50** [comment]
A Small UP please

**Comment by ddurieux** [comment]
yes it's not possible, I think adding code to manage it, but not simple

**Comment by FRANCOIS50** [comment]
HI, Do you have an ID if this problem will be solved?
